benchmarks/linear_algebra/blas/level1/nrm2/nrm2_generator.cpp

```diff
@@ -0,0 +1,68 @@
+
+
+#include <tiramisu/tiramisu.h>
+#include "benchmarks.h"
+#include "math.h"
+/*
+  IMPLEMENTATION OF THE NMR2 FUNCTION IN TIRAMISU 
+  
+   out = sqrt(x*'x) 
+
+   where : 
+   x : is a size N vector and 'x is the transpose of x 
+   
+*/
+
+
+
+using namespace tiramisu;
+
+int main(int argc, char **argv)
+{
+    tiramisu::init("nrm2");
+
+    // -------------------------------------------------------
+    // Layer I
+    // -------------------------------------------------------
+
+    constant NN("N", expr(N));
+
+    var i("i", 0, NN);
+    input x("x", {"i"}, {NN},p_float64);
+    computation S_init("S_init", {}, expr(cast(p_float64,0)));
+    computation S("S", {i},p_float64);
+    S.set_expression( S(i-1) + x(i)*x(i));
+    computation result("result", {},p_float64);
+    result.set_expression(cast(p_float64,expr(o_sqrt,S(NN),p_float64))); 
+     
+    
+
```

Comment:
This code has a lot of issues related to coding style, can you please address them and update your pull request?

Reply:
Yes of course.

Reply:
> Your code has merging conflicts. Can you please look at them and fix them?
Salem Sir, 
I've resolved the conflicts and it 's clean now